scripts/make_glibc_and_sysroot.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
```

Comment:
if this file replaces other scripts functionality should those scripts be removed?

Reply:
This needs some planning. `lindtool.sh` depends on those scripts, and we depend on `lindtool.sh` via `wasmtestreport.py`. I will make a proposal to untangle these local build tools.